sessionctx/variable/tidb_vars.go

```diff
@@ -445,7 +445,7 @@ const (
 	DefIndexSerialScanConcurrency      = 1
 	DefIndexJoinBatchSize              = 25000
 	DefIndexLookupSize                 = 20000
-	DefDistSQLScanConcurrency          = 15
+	DefDistSQLScanConcurrency          = 6
```

Comment:
It seems `DefDistSQLScanConcurrency` changed from 15 to 6 will make `physical_plan_test.TestAggEliminator` failed. here is the error log:
```sh
FAIL: physical_plan_test.go:413: testPlanSuite.TestAggEliminator

physical_plan_test.go:444:
    c.Assert(core.ToString(p), Equals, output[i].Best, Commentf("for %s", tt))
... obtained string = "IndexMergeJoin{TableReader(Table(t))->TableReader(Table(t))}(test.t.a,test.t.a)->Limit->StreamAgg"
... expected string = "MergeInnerJoin{TableReader(Table(t))->TableReader(Table(t))}(test.t.a,test.t.a)->Limit->StreamAgg"
... for select max(a) from (select t1.a from t t1 join t t2 on t1.a=t2.a) t

```